src/Mapping/Annotation/Slug.php

```diff
@@ -41,4 +45,40 @@ final class Slug extends Annotation
     public $handlers = [];
     /** @var string */
     public $dateFormat = 'Y-m-d-H:i';
+
+    /**
+     * @param string[]      $fields
+     * @param SlugHandler[] $handlers
+     */
+    public function __construct(
+        array $data = [],
+        array $fields = [],
+        bool $updatable = true,
+        string $style = 'default',
+        bool $unique = true,
+        ?string $unique_base = null,
```

Comment:
I'd try to use camel case where possible.

Reply:
Here the problem is that this name is used also for annotations (as name parameter), so we need to keep it for now. 

I agree about using camel case, but that can be done in another PR deprecating the snake case.